dev_guide/builds.adoc

```diff
@@ -385,6 +384,31 @@ sourceStrategy:
 You can also manage environment variables defined in the `*BuildConfig*` with the
 xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command.
 
+[[adding-secrets-to-source-strategy-build-configs]]
+=== Adding Secrets to Source Strategy Build Configurations
+
+Add a secret to your build configuration so that it can access a private
+repository.
+
+. Create a new {product-title} project.
+
+. xref:../dev_guide/secrets.adoc#dev-guide-secrets[Create a secret] that contains
+credentials for accessing a private repository.
+
```

Comment:
IMO, this is not precise enough. The US can add secrets for private repo including both source code repo (git) and image registry repo (docker hub). The former is so-called Source Secret, the latter is Pull/Push Secret. So this statement would be better "Create a secret that contains credentials for accessing a private source code repository" to avoid obscure